src/runtime/audience-action-local-flow.ts

```diff
@@ -647,15 +701,22 @@ export class AudienceActionLocalFlow implements AudienceActionFlow {
   }
 
   private getCloseButtonHtml_(html: string) {
+    if (
+      !this.params_.isClosable ||
```

Comment:
Would it be possible to override this value at the top (probably in the constructor?) so that it's the single source of the truth throughout the action flow? There are other code referecning this fields and the outcome can be unexpected if `this.params_.isClosable` = true but close button is actually not rendered.

Reply:
I don't think we want to do that. We only want the initial prompt to be unclosable, but for it to otherwise act like a closable prompt. For example, if an ad view is canceled causing the `rewardedSlotClosed` callback to be issued, we want to allow the page to be unlocked. If we override `isClosable` in the constructor it would keep it locked.

Reply:
Ah, got it. Thanks for clarification. It's really hard to understand from code that closability between initial and other prompts can differ. Can we make it easy to understand by declaring variables with clear name?

Reply:
Moved the condition into a named variable.